ci/pre-prod/deploy_demo_playbook.yml

```diff
@@ -136,6 +136,21 @@
           - "{{ firecrest_dir }}/deploy/test-build/environment/keys/user-key:/user-key:ro"
           - "{{ firecrest_dir }}/deploy/demo/ssl:/ssl"
 
+    - name: Reservations
+      community.general.docker_container:
+        name: reservations
+        image: "{{ docker_registry_host }}/reservations:{{build_tag}}"
+        env_file: "{{ firecrest_dir }}/deploy/demo/common/common.env"
+        networks:
+              - name: firecrest-internal
+                ipv4_address: "192.168.220.8"
+        published_ports:
+          - "5005:5005"
+        volumes:
+          - "{{ firecrest_dir }}/deploy/demo/logs/firecrest:/var/log:delegated"
+          - "{{ firecrest_dir }}/deploy/test-build/environment/keys/user-key:/user-key:ro"
```

Comment:
just double-checking, is `test-build` path ok here? With my limited understanding so far I was expecting only stuff from `demo` path for `pre-prod` stuff.

Reply:
Yes, because ssh keys `demo` are the same that the ones that appear on `test-build` so we don't duplicate them